### PR TITLE
Kb rwa giz

### PR DESCRIPTION
**Pull request by fheslouin**
Can you review and merge ? 

Thanks

### CODE REVIEW

**Comment by yohanboniface** [comment]
Seems the statics in the package are still loaded from the original domain. Here is what I have when I test it:

    <link rel="stylesheet" type="text/css" media="all" href="http://www.24hdansuneredaction.com/wp-content/themes/24hdansuneredaction/base.css">

File: ideascube/conf/kb_rwa_giz.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+"""KoomBook conf"""
+from .kb import *  # noqa
+
+LANGUAGE_CODE = 'en'
+IDEASCUBE_NAME = 'GIZ'
```
Comment (bochecha) [inline ideascube/conf/kb_rwa_giz.py]: Did you know we don't need to specify the `IDEASCUBE_NAME` in the settings files any more? 😄

Comment (fheslouin) [inline ideascube/conf/kb_rwa_giz.py]: Ok, but what is the default name then ?

Comment (fheslouin) [inline ideascube/conf/kb_rwa_giz.py]: I'm using set server name currently in the playbook, however the name picked up to set the var is the project name which is not really nice...I'll remove IDEASCUBE_NAME

People will have to find their way to the admin page to change the displayed name...

Comment (bochecha) [inline ideascube/conf/kb_rwa_giz.py]: > I'm using set server name currently in the playbook, however the name picked up to set the var is the project name which is not really nice...

Well, but that could be made nicer with a separate variable, set to a prettier name, no?

Comment (fheslouin) [inline ideascube/conf/kb_rwa_giz.py]: Yep but we are not always aware of the name wished by the end-user :/
See #634

**Pull request merged**